readme.md

```diff
@@ -27,8 +26,6 @@ This is a collection of unofficial skins for Grande Omega. It changes and adds s
 
 Download the repo using the "**↓** Code" button and select the ZIP file.
 
-Make a **copy** of the resource folder in the Grande Omega folder.
```

Comment:
Je hebt hier een instructie weg gehaald waar later in de ReadMe wel naar gerefereerd word